napari/_qt/menus/__init__.py

```diff
@@ -2,14 +2,12 @@
 from .file_menu import FileMenu
 from .help_menu import HelpMenu
 from .plugins_menu import PluginsMenu
-from .view_menu import ViewMenu
```

Comment:
Just confirming once you've done app-model for all menus all the Menu imports here will go away - but this PR only does ViewMenu

Reply:
correct